ai-services/cmd/ai-services/cmd/bootstrap/bootstrap.go

```diff
@@ -15,13 +16,23 @@ func BootstrapCmd() *cobra.Command {
 		Short: "Initializes AI services infrastructure",
 		Long: `Bootstrap and configure the infrastructure required for AI services.
 
-The bootstrap command prepares and validates the environment needed
+The bootstrap command configures and validates the environment needed
 to run AI services on Power11 systems, ensuring prerequisites are met
 and initial configuration is completed.
 
 Available subcommands:
-  validate   - Check system prerequisites and configuration
-  configure  - Configure and initialize the AI services infrastructure`,
+  configure - Configure performs below actions
+  • Installs podman on host if not installed
+  • Runs servicereport tool to configure required spyre cards
+  • Initializes the AI services infrastructure
+
+  validate - Checks below system prerequisites
+  • root user
```

Comment:
nit, for uniformity. Can be taken later
```suggestion
  • Root user
```